compile.sh

```diff
@@ -7,7 +7,14 @@ mkdir -p build/gui/mac/amd/Tonutils\ Proxy.app/Contents/MacOS
 CGO_CFLAGS=-mmacosx-version-min=10.9 CGO_LDFLAGS=-mmacosx-version-min=10.9 CGO_ENABLED=1 GOOS=darwin GOARCH=amd64 go build -ldflags '-w -s' -o build/gui/mac/amd/Tonutils\ Proxy.app/Contents/MacOS/tonutils-proxy cmd/proxy-gui/main.go
 
 echo "Building WINDOWS GUI AMD"
-CGO_ENABLED=1 GOOS=windows GOARCH=amd64 go build -o build/tonutils-proxy-gui.exe cmd/proxy-gui/main.go
+p=$(pwd)
+# To start script need to declare your own LIBS env variable
+LIBS="C:\Users\alexb\libs"
```

Comment:
lets remove it from here, can be set during build, just add a comment where it should point to